packages/samurai.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Samurai < Package
+  description 'Samurai is a ninja compatible build tool written in C.'
+  homepage 'https://github.com/michaelforney/samurai'
+  version '1.2'
+  license 'Apache-2.0 and MIT'
+  compatibility 'all'
+  source_url 'https://github.com/michaelforney/samurai/releases/download/1.2/samurai-1.2.tar.gz'
+  source_sha256 '3b8cf51548dfc49b7efe035e191ff5e1963ebc4fe8f6064a5eefc5343eaf78a5'
+
+  def self.patch
+    system "sed -i 's:PREFIX=/usr/local:PREFIX=#{CREW_PREFIX}:' Makefile"
+    system "sed -i 's:MANDIR=\$(PREFIX)/share/man:MANDIR=#{CREW_MAN_PREFIX}:g' Makefile"
```

Comment:
Could we set these in the build section instead?  Something like:
```
system "PREFIX=#{CREW_PREFIX} MANDIR=#{CREW_MAN_PREFIX} \
            make CFLAGS='-pipe -O2 -flto=auto -fuse-ld=gold' \
            LDFLAGS='-flto=auto -fuse-ld=gold' \
            CC='#{CREW_TGT}-gcc'"
```

Reply:
Not with the way the makefile is written, no. I already tried that.

Reply:
I would be surprised if it didn't work.

Reply:
Becuase it's `PREFIX=/usr/local` not `PREFIX:=/usr/local` it doesn't work. Same with MANDIR

Reply:
Ah, that makes sense now.